.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: node_js
 node_js:
   - "8"
-  - "10"
```

Comment:
Is it ok to downgrade the version? which has been used as the build environment?

Reply:
just moved line.
Build tests of both nodes 8 and 10.